### DISCUSSION TITLE
Adding support for  UniFormer 支持 UniFormer

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#277] [MMSegmentation]
# Adding support for  UniFormer 
 支持 UniFormer 

[https://github.com/Sense-X/UniFormer/tree/main/semantic_segmentation](https://github.com/Sense-X/UniFormer/tree/main/semantic_segmentation) 

## Task Information

**Repository**：MMSegmentation

**Difficulty**：Hard

**Points**：50

**Estimated Time Consumption**：28 days

## Task Status

**Claimant's Github ID**：jinxianwei

**Claim Time**：2023年09月04日

**PR Link**：https://github.com/open-mmlab/mmsegmentation/pull/3362

## Points to Note

To claim the task, please reply `Claim the mission` in the comment section. After claiming the task, please add our assistant bot called MeowMewo on WeChat at: `openmmlabwx`.

After completing the task, please submit a PR with the title starting with [MMSIG]. After submission, please reply with your PR link in the comments section of this Discussion. We will review your work as soon as possible.

在评论区回复 `Claim the mission` 领取任务，领取任务后请及时添加小助手喵喵微信: `openmmlabwx`

完成任务后提交相关链接（如是PR，标题请以[MMSIG]开头）回复到本Disucssion评论区，我们会第一时间审核您的贡献。

**Comment by jinxianwei** [comment]
Claim the mission

**Comment by jinxianwei** [comment]
https://github.com/open-mmlab/mmsegmentation/pull/3362

**Reply by vansin** [reply]
good
